docs-markdown/src/controllers/master-redirect-controller.ts

```diff
@@ -62,6 +72,33 @@ function showStatusMessage(message: string) {
     output.show();
 }
 
+export async function sortMasterRedirectionFile() {
+    const editor = window.activeTextEditor;
+    if (!editor) {
+        postWarning("Editor not active. Abandoning command.");
+        return;
+    }
+
+    const folder = workspace.getWorkspaceFolder(editor.document.uri);
+    if (folder) {
+        const file = tryFindFile(folder.uri.fsPath, redirectFileName);
+        if (!!file && fs.existsSync(file)) {
+            const jsonBuffer = fs.readFileSync(file);
+            const redirects = JSON.parse(jsonBuffer.toString()) as IMasterRedirections;
+            if (redirects && redirects.redirections && redirects.redirections.length) {
+
+                redirects.redirections.sort((a, b) => {
+                    return naturalLanguageCompare(a.source_path, b.source_path);
+                });
+
+                fs.writeFileSync(
+                    file,
+                    JSON.stringify(redirects, ["redirections", "source_path", "redirect_url", "redirect_document_id"], 4));
+            }
+        }
+    }
```

Comment:
If `redirectFileName` does not exist, can we raise a notification?

Reply:
Like a `postWarning`?

Reply:
Done